crowdin.yaml

```diff
@@ -1,19 +1,3 @@
-project_identifier: django-girls-tutorial
```

Comment:
`project_identifier` is used for API integration, actually We could leave it here, as it does no harm. Do You want me to leave it here (as I believe, we don't use API integration with Crowdin)?

See: https://support.crowdin.com/api/api-integration-setup/